artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/logger/ActiveMQAMQPProtocolLogger.java

```diff
@@ -32,19 +32,25 @@ public interface ActiveMQAMQPProtocolLogger {
    @LogMessage(id = 111000, value = "Scheduled task can't be removed from scheduledPool.", level = LogMessage.Level.WARN)
    void cantRemovingScheduledTask();
 
-   @LogMessage(id = 111001, value = "\n*******************************************************************************************************************************" +
-      "\nCould not re-establish AMQP Server Connection {} on {} after {} retries" +
-      "\n*******************************************************************************************************************************\n", level = LogMessage.Level.WARN)
+   @LogMessage(id = 111001, value = """
+      *******************************************************************************************************************************
```

Comment:
Did you mean to remove the deliberate newline at the start?
(same for others)

Reply:
Fixed.